src/CTBase.jl

```diff
@@ -14,7 +14,8 @@ module CTBase
 # using
 import Base
 using DocStringExtensions
-using ForwardDiff: jacobian, gradient, ForwardDiff # automatic differentiation
```

Comment:
In general, `using MyPackage: name1, name2, name3` loads three names from `MyPackage`. Here I want to load `ForwardDiff` but I don't want any names inside it because they would collide with the names in DifferentiationInterface, so I just use `name1 = ForwardDiff` itself.
It's essentially the same as `import ForwardDiff`, but I'm used to coding in [BlueStyle](https://github.com/JuliaDiff/BlueStyle?tab=readme-ov-file#module-imports) and the automatic formatter prefers `using ForwardDiff: ForwardDiff` in such cases. I'll make the change though if you prefer the other way.

Reply:
Oh; fine, then. Was not aware of that BlueStyle convention. As I had just read yout tip [in there](https://gdalle.github.io/DifferentiationInterface.jl/DifferentiationInterface/stable/tutorial1/) on this, I was actually surprised not to see an `import`. Let's be blue 🙂.

Reply:
I changed it in the latest commit to be less blue ;) `import` is the thing that most people do, so it's fine.

I just tend to avoid it because if you write `import MyPackage: f1` then define `f1(x) = ...` you may accidentally redefine or overwrite a method. With `using MyPackage: f1`, you have to be intentional to redefine things because you need `MyPackage.f1(x) = ...`.